holoclean.py

```diff
@@ -214,16 +219,22 @@ def repair_errors(self, featurizers):
             print('Time to collect inferred values: %.2f secs' % time)
         status, time = self.ds.get_repaired_dataset()
         print(status)
+        if self.env['verbose']:
```

Comment:
Shouldn't we be checking for the 'printfw' flag (instead of verbose)?

Reply:
In this case, where is the 'printfw' flag used?

Reply:
it's due to inconsistent naming in flag name and dest name as Mina suggested. I've changed both of them to printfw